packages/salesforcedx-vscode-core/src/commands/forceProjectCreate.ts

```diff
@@ -127,14 +127,6 @@ export class SelectProjectTemplate
         'force_project_create_analytics_template'
       )
     ];
-    if (sfdxCoreSettings.getFunctionsEnabled()) {
```

Comment:
don't we need to remove `projectTemplateEnum.functions`?

Reply:
+1, I think we'll also need to remove [this reference](https://github.com/forcedotcom/salesforcedx-vscode/blob/0289155546ba1d3923a99e07189bef8797d5f2d0/packages/salesforcedx-vscode-core/src/commands/forceProjectCreate.ts#L143)

Reply:
Good call! Will take care of these.